### ISSUE TITLE
Compiling

### ISSUE THREAD

**Issue by Codex-NG17** [hjnilsson/rme#193]
You need to update that wiki compiling guide because if you are on a modern day version of linux some of those packages/libraries don't exist.

**Comment by Elime1** [comment]
What linux dist and version are you running?

**Comment by Codex-NG17** [comment]
I am running Linux Mint 18.1 Cinnamon x64. See most people skip the read me and go straight to the wiki, especially if they are accustomed to large scale projects. You have information both in the readme (which most never read) and the wiki for linux that contradicts the wiki.

Ubuntu & Linux Mint are very similar they use relatively the same packages, repositories and commands just have a different interface and are preferred over the other.

Unfortunately if you are compiling on anything but Ubuntu libwxgtk3.0-0-dbg will halt the installation of any additional packages because it won't be able to find it. 

To summarize, if the readme instructions work then maybe you should update the wiki so it reflects the readme and there is no confusion. Thanks

**Comment by Codex-NG17** [comment]
Just updating that the readme instructions do work when trying to compile on Linux Mint, however the wiki does not.

**Comment by Elime1** [comment]
Updated the wiki and added a link to it in the readme a60d0aa.
